src/components/Text.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import '../css/text.css';
 
-function Text({textTitle, textPara, button}){
-		return (
-			<div className="text">
-				<h4 className="fade-in-text">{textTitle}</h4>
-				<p className="fade-in-text">
-					{textPara}
-				</p>
-				<button className="fade-in-text">{button} </button>
-			</div>
-		);
+function Text({title, textPara, buttonCopy}){
+	return (
+		<div className="text">
+			<h4 className="fade-in-text">{title}</h4>
```

Comment:
Indentation is still rough, but don't worry about it for now. We'll meet soon about prettier settings.